landing-pages/site/content/en/blog/implementing-stable-API-for-Apache-Airflow.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Implementing Stable API for Apache Airflow"
+linkTitle: "Implementing Stable API for Apache Airflow"
+author: "Ephraim Anierobi"
+twitter: "ephraimbuddy"
+github: "ephraimbuddy"
+description: "An Outreachy intern's progress report on contributing to Apache Airflow REST API."
+tags: ["REST API"]
+date: "2020-07-19"
+---
+
+My [Outreachy internship](https://outreachy.org) is coming to its ends which is also the best time to look back and
+reflect on the progress so far.
+
+The goal of my project is to Extend and Improve the Apache Airflow REST API. In this post,
+I will be sharing my progress so far.
+
+We started a bit late implementing the REST API because it took time for the OpenAPI 3.0
+specification we were to use for the project to be merged. Thanks to [Kamil](https://github.com/mik-laj),
+who paved the way for us to start implementing the REST API endpoints. Below are the endpoints I
+implemented and the challenges I encountered, including how I overcame them.
+
+### Implementing The Read-Only Connection Endpoints
+The [read-only connection endpoints](https://github.com/apache/airflow/pull/9095) were the first endpoint I implemented. Looking back,
+I can see how much I have improved.
+
+I started by implementing the database schema for the Connection table using Marshmallow 2.
+We had to use Marshmallow 2 because Flask-AppBuilder was still using it and Flask-AppBuilder
+is deeply integrated to Apache Airflow. This means I had to unlearn Marshmallow 3 that I have
+ been studying before this realization and thankfully, I started using Marshmallow 2 in no time.
```

Comment:
Consider adjusting the last part of this sentence to explain why you were able to use it in no time - perhaps something like 

"...before this realization, but thankfully, Marshmallow 3 isn't too different, so I was able to start using Marshmallow 2 in no time."

You can swap out "isn't too different" with whatever reason is true for you - "the docs are detailed" "my mentor helped" or whatever it is 😄